### DISCUSSION TITLE
Adherent/Volunteer feature

### DISCUSSION THREAD

**Discussion by mickahell** [ESNFranceG33kTeam/CosmoAppy#14] [Ideas]
## Definition
**Adherent**
De mon coté un adherent est défini comme ceci : 
- first name
- last name
- email
- date of birth
- esncard --> (si pas de carte, laissé vide)
- student --> oui/non
- date of the day --> automatique pour RGPD

**Bénévole**
Et un bénévole pour moi est un adherent++, ce qui signifie c'est un adhérent avec ce qui suit en plus :
- (tout ce qui vient d'adherent)
- status --> actif / alumnis
- bureau --> oui/non

---
What Do You Thing ?
@yoanpom

**Comment by yoanpom** [comment]
Hey, pour nous l'adhérent c'est juste :

  * first name
  * last name
  * mail
  * date of birth
  * university
  * country (pas forcément la nationalité mais pays d'origine, c'est du
    déclaratif)

voilà :)

*Yoan Pompet*
Retrouvez moi aussi sur Linkedin <https://www.linkedin.com/in/ypompet/> 
et Twitter <https://twitter.com/YoanPom>
Le 25/08/2022 à 07:56, Michaël a écrit :

**Reply by mickahell** [reply]
Pas besoin de savoir si c'est un étudiant pour les buddy family ? 

Un non adhérent peut-il acheter une carte ESN ?
